package/MDAnalysis/coordinates/chemfiles.py

```diff
@@ -375,7 +389,17 @@ def _timestep_to_chemfiles(self, ts):
         if ts.has_velocities:
             frame.add_velocities()
             frame.velocities[:] = ts.velocities[:]
-        frame.cell = chemfiles.UnitCell(*ts.dimensions)
+
+        lengths = ts.dimensions[:3]
+        angles = ts.dimensions[3:]
+        if angles[0] == 0.0 and angles[1] == 0.0 and angles[2] == 0.0:
+            angles = [90, 90, 90]
+
+        if chemfiles.__version__.startswith("0.9"):
+            frame.cell = chemfiles.UnitCell(*lengths, *angles)
```

Comment:
In this case, imho it's acceptable for now to just not cover it. Adding various versions down in CI is going to eventually become a pain.

Can you open an issue about covering this so we don't forget? I was working on a weekly cron CI job somewhere else, I can probably just add a workflow that installs older versions of chemfiles.